packages/vscode-extension/src/OperationResult.ts

```diff
@@ -0,0 +1,16 @@
+export default class OperationResult {
```

Comment:
Also, for me this is not a class, its a type or an interface. The constructor is never called.
I'd add it in a new folder, like types/OperationResult.d.ts since its used to define a type that is the result operation from node-cli in javascript

Reply:
I agree this should ne an interface, not a class. I don't think the file should be .d.ts though, defintion files are for JS libraries to be used in TS code. I would convert this to interface (no constuctor), leave filename as it is and move it to commands folder for now - as it's only used by command classes at this point.

Reply:
Done!